app/assets/javascripts/controllers/live_metrics/util/metrics-http-factory.js

```diff
@@ -27,11 +27,8 @@ angular.module('miq.util').factory('metricsHttpFactory', function() {
 
       angular.forEach(dash.items, getLatestData);
 
-      if (dash.items.length > dash.max_metrics) {
-        dash.filterConfig.resultsCount = __("Showing first") + " " + dash.max_metrics;
-      } else {
-        dash.filterConfig.resultsCount = dash.items.length;
-      }
+      dash.pages = (data.pages > 0) ? data.pages : 1;
+      dash.filterConfig.resultsCount = __("Page ") + data.page + __(" Of ") + dash.pages + __(", Found ") + data.items;
```

Comment:
@yaacov This is an i18n bug, can you fix please?

Not only does this lead to "Page 1 Of 3" in english, but it will lead to nonsense in any other language, we need to keep the whole phrase together for i18n.

So.. I suggest..

```sprintf(__("Page %d of %d, found %d results"), data.page, dash.pages, data.items);```